doc/user_guide/Streaming_Plots.rst

```diff
@@ -0,0 +1,6 @@
+Streaming Plots
+_______________
+
+.. notebook:: holoplot ../../examples/user_guide/Streaming_Plots.ipynb
+    :skip_execute: True
+
```

Comment:
Is there some reason to commit .rst files like this to the repo?  Shouldn't they just be auto-generated?

Reply:
Yes, it's the reason I filed https://github.com/pyviz/nbsite/issues/43, this notebook should not be run and therefore this file declares ``:skip_execute: True``.